indico/modules/rb_new/controllers/backend/bookings.py

```diff
@@ -153,3 +154,12 @@ def _process(self):
         booking_details['occurrences'] = occurrences
         booking_details['date_range'] = date_range
         return jsonify(booking_details)
+
+
+class RHBookingEventData(RHBookingDetails):
```

Comment:
what about adding a `RHBookingBase` that contains the `_process_args` that's currently in `RHBookingDetails` and then use that class as a base for both `RHBookingDetails` and `RHBookingEventData`?

Reply:
Thought of it, but thought it was an overkill.